examples/howto/plot_connectivity.py

```diff
@@ -95,14 +97,14 @@ def get_network(probability=1.0):
 
 
 net_all = get_network()
-dpl_all = simulate_dipole(net_all, n_trials=1)
+dpl_all = simulate_dipole(net_all, n_trials=1, postproc=True)
```

Comment:
oops didn't notice this. If we deprecate it, then we shouldn't use it because our examples will throw warnings then. Also, you'll have to deal with this next release when we remove this parameter.

Reply:
Yeah I can fix that real quick...